src/components/Widget/IntegrationsWidget.scss

```
@@ -33,8 +33,18 @@
+  .empty-state-icon {
+    color: var(--pf-t--global--text--color--regular);
+
+    img {
+      height: 36px;
+      width: auto;
+    }
+
+    svg {
+      height: 36px;
+      width: auto;
+      fill: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)